emerlin2caom2/main_app.py

```diff
@@ -166,16 +186,21 @@ def measurement_set_metadata(self, observation, ms_dir, pickle_dict, plane_id):
         # This one isn't working quite right yet-- see obs_reader_writer.py
         # plane.polarization.polarization_states = pol_states
 
-        provenance = Provenance(pickle_dict['pipeline_path'])
+        provenance = Provenance(pickle_dict['pipeline_path'].split('/')[-1])
```

Comment:
Fails xml validation for caom2/plane.py where no name exists, is this another where my version of data product doesn't contain an artifact/plane object that Michael's does, and we need a caveat 'if' statement? 

> Traceback (most recent call last):
>   File "run_script.py", line 4, in <module>
>     a.build_metadata()
>   File "/Users/user/repos/emerlin2caom-uploading_in_python/emerlin2caom2/main_app.py", line 311, in build_metadata
>     self.measurement_set_metadata(observation, self.ms_dir_main, pickle_obj, ms_plane_id)
>   File "/Users/user/repos/emerlin2caom-uploading_in_python/emerlin2caom2/main_app.py", line 189, in measurement_set_metadata
>     provenance = Provenance(pickle_dict['pipeline_path'].split('/')[-1])
>   File "/Users/user/miniconda3/envs/py38/lib/python3.8/site-packages/caom2/plane.py", line 843, in __init__
>     raise AttributeError("No name provided")
> AttributeError: No name provided

Reply:
SOLVED.